tests/test/tuples.rs

```diff
@@ -82,13 +82,41 @@ fn tuples_are_sized() {
         } yields {
             "Unique; substitution [], lifetime constraints []"
         }
+
+        goal {
+            exists<T> { (T, u8): Sized }
+        } yields {
+            "Unique; for<?U0> { substitution [?0 := ^0.0], lifetime constraints [] }"
+        }
+
+        goal {
+            forall<T> { (T, u8): Sized }
+        } yields {
+            "Unique; substitution [], lifetime constraints []"
+        }
+
+        goal {
+            forall<T> { (u8, T): Sized }
```

Comment:
If we instead made `Sized` `non_enumerable`, this would give `Ambiguous`, right?

Reply:
No, it's still no possible solution (since this is a `forall`, I think).